packages/styles/components/_c.accordion.scss

```diff
@@ -0,0 +1,89 @@
+.mc-accordion {
+  $parent: &;
+
+  background-color: $color-grey-000;
+
+  @include set-font-face();
+
+  &__label {
+    @include set-font-scale('06', 'm');
+    @include set-font-face('semi-bold');
+
+    color: $color-grey-900;
+    cursor: pointer;
+    position: relative;
+    padding: $mu125 $mu050;
+    display: flex;
+    align-items: center;
+    background-image: url(inline-icons('control-more-24',$color-grey-900));
+    background-position: right 1rem center;
+    background-repeat: no-repeat;
+    background-size: $mu150;
+
+    &:hover {
+      background-color: $color-grey-100;
+    }
+
+    &:focus {
+      @include set-focus-styles-during();
+    }
+
+    &--no-padding {
+      padding-left: 0;
+    }
+  }
+
+  &__picto {
+    width: $mu200;
+    height: $mu200;
+    margin-right: $mu100;
+  }
+
+  &__content {
+    height: auto;
+    max-height: 0;
+    overflow: hidden;
+    padding-left: $mu050;
+    transition: max-height 0.3s ease-out, padding-bottom 0.3s linear;
+    border-bottom: get-border(s) solid $color-grey-600;
+    color: $color-grey-800;
+  }
+
+  &__trigger {
+    opacity: 0;
+    position: absolute;
+
+    &:checked {
+      ~ #{$parent}__label {
+        background-image: url(inline-icons('control-less-24', $color-grey-900));
+      }
+      ~ #{$parent}__content {
+        max-height: 100vh;
+        padding-bottom: $mu150;
+      }
+    }
+
+    &:disabled {
+      ~ #{$parent}__label {
+        background-color: $color-grey-100;
+        background-image: url(inline-icons('control-more-24', $color-grey-600));
+        color: $color-grey-600;
```

Comment:
Better to use accessible variable : 
~~color: $color-grey-600;~~ => `$color-font-light`